docs/user/citing.md

```diff
@@ -49,6 +54,7 @@ url={}
 
 - Release 1.0.0 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3727270.svg)](https://doi.org/10.5281/zenodo.3727270)
 - Release 1.1.0 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3978643.svg)](https://doi.org/10.5281/zenodo.3978643)
+- Release 1.2.0 [![DOI](add new zenodo link here)]()
```

Comment:
```suggestion
```

Just removing this change as we might make the version number different but thanks for the point thins out will add it to my check list